### PR TITLE
Allow creating threads on user (needed for an MVP)

### PR DESCRIPTION
**Pull request by kulla**
Closes https://github.com/serlo/serlo.org-database-layer/issues/126

@elbotho @inyono How is first can make the review :smile:

### CODE REVIEW

**Comment by elbotho** [comment]
first ;)
looks good, but @inyono should still review this… 
especially the sql.

File: src/thread/model.rs [hunk 1]
```diff
@@ -390,7 +391,7 @@ impl Threads {
             .save(&mut transaction)
             .await?;
 
-        if payload.subscribe {
+        if payload.subscribe && uuid_data.discriminator != "user" {
```
Comment (inyono) [inline src/thread/model.rs:394]: Does this make sense? Shouldn't I still get notifications for threads created at user profiles (or wherever those are displayed)?

File: src/thread/model.rs [hunk 2]
```diff
@@ -424,36 +425,49 @@ mod tests {
...
-                object_id: 1565,
-                user_id: 1,
-                subscribe: true,
-                send_email: false,
-            },
-            &mut transaction,
-        )
-        .await
-        .unwrap();
+        for object_id in [1565, 1] {
```
Comment (inyono) [inline src/thread/model.rs:428]: Rather have those in two tests since they test different things.

**Comment by kulla** [comment]
@inyono Thx for the feedback. Added both suggestions. Ready for merge.

**Review by inyono — APPROVED** [review]

**Pull request merged**